src/components/global/Footer.jsx

```diff
@@ -36,34 +38,38 @@ const legalLinks = [
 const socialMediaLinks = [
   {
     href: 'https://github.com/codeforpdx/',
-    icon: <FaGithub size={45} />
+    icon: <FaGithub size={45} />,
+    ariaLabel: 'Github'
   },
   {
     href: 'https://www.linkedin.com/company/code-pdx/',
-    icon: <FaLinkedin size={45} />
+    icon: <FaLinkedin size={45} />,
+    ariaLabel: 'Linked In'
   },
   {
     href: 'https://www.meetup.com/Code-for-PDX/',
-    icon: <FaMeetup size={45} />
+    icon: <FaMeetup size={45} />,
+    ariaLabel: 'Meetup.com'
   },
   {
     href: 'https://discord.gg/x6b573et',
-    icon: <FaDiscord size={45} />
+    icon: <FaDiscord size={45} />,
+    ariaLabel: 'Discord'
   }
 ];
 
 // renders socialMediaLinks for placement in stack below
-const renderSocialLinks = socialMediaLinks.map(({ href, icon }) => (
+const renderSocialLinks = socialMediaLinks.map(({ ariaLabel, href, icon }) => (
   <Link
     component={ReactRouterLink}
     key={href}
     href={href}
+    aria-label={`Check us out on ${ariaLabel}`}
     target="_blank"
     rel="noopener"
-    color="#000"
     sx={{
+      color: '#000',
       display: 'flex',
-      alignItems: 'center',
       padding: '25px',
```

Comment:
make this `padding` in `renderSocialLinks` only a `py: '25xp'`, then see comment on line 189.